starter-code/app.js

```diff
@@ -54,5 +47,10 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+const celebrityRoutes = require('./routes/celebrities');
+app.use('/', celebrityRoutes);
```

Comment:
You can also decide to mount the route on top of a path. 

For example: `app.use('/celebrities', celebrityRoutes);`. 

In this way you won't need to prefix every route in the routes/celebrities.js file with the _/celebrities_ path

Reply:
Really good work overall!